### PR TITLE
feat(usecase): add Business Logic to upload document

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Connected usecase upload document functionality with business logic

## Why

User can upload document for particular usecase if not exist.

## Issue

N/A

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: src/components/overlays/EditUsecase/index.tsx [hunk 1]
```diff
@@ -25,37 +28,64 @@ import {
...
         {...{
-          title: t('content.usecaseParticipation.editUsecase.title'),
-          intro: t('content.usecaseParticipation.editUsecase.description'),
+          title: t('content.usecaseParticipation.editUsecase.title').replace(
+            '{usecaseName}',
+            credentialType
+          ),
+          intro: t(
+            'content.usecaseParticipation.editUsecase.description'
+          ).replace('{usecaseName}', credentialType),
```
Comment (manojava-gk) [inline src/components/overlays/EditUsecase/index.tsx:88]: There is a concept called interpolation. no need to use replace. check checklistOverlay section.
https://www.i18next.com/translation-function/interpolation

**Review by manojava-gk — COMMENTED** [review]
Small finding

**Review by oyo — CHANGES_REQUESTED** [review]
agree to Mano's remark - there is a better way to replace values in i18n.

**Review by manojava-gk — APPROVED** [review]
lftm

**Review by evegufy — APPROVED** [review]

**Pull request merged**